src/tests/ftest/cart/util/cart_utils.py

```diff
@@ -38,6 +38,8 @@
 import cart_logparse
 import cart_logtest
 
+from write_host_file import write_host_file
```

Comment:
(pylint-import-error) Unable to import 'write_host_file' (invalid syntax (<string>, line 65))

Reply:
@mallove79 , can you fix this?

Reply:
I'll try to replicate this in Visual Studio pylint. It seems it requires a change to @daosbuild1 's `PYTHONPATH`, but where is `PYTHONPATH` set for @daosbuild1 pylint?

Reply:
I'm blocked with the `import-error` pylint message. The same python import line `from write_host_file import write_host_file` is in nine other `ftest` files:

```
$ grep -Eir '^from write_host' src
src/tests/ftest/nvme/nvme_fragmentation.py:from write_host_file import write_host_file
src/tests/ftest/nvme/nvme_pool_capacity.py:from write_host_file import write_host_file
src/tests/ftest/osa/osa_online_drain.py:from write_host_file import write_host_file
src/tests/ftest/osa/osa_online_reintegration.py:from write_host_file import write_host_file
src/tests/ftest/osa/osa_online_extend.py:from write_host_file import write_host_file
src/tests/ftest/osa/osa_online_parallel_test.py:from write_host_file import write_host_file
src/tests/ftest/util/apricot/apricot/test.py:from write_host_file import write_host_file
src/tests/ftest/util/job_manager_utils.py:from write_host_file import write_host_file
src/tests/ftest/io/seg_count.py:from write_host_file import write_host_file
```

But maybe the flag was only raised in the case of `cart_utils.py` because it's a utility module importing another utility module?

Reply:
Adding space after comma in `write_host_file.py` resolves pylint message: https://github.com/daos-stack/daos/pull/3603/commits/4426d8ef00116a6e678205d0bb9763f6628eab19.